### DISCUSSION TITLE
Places to post about pollinations 0.1 milestone release

### DISCUSSION THREAD

**Discussion by voodoohop** [pollinations/pollinations#121] [Pollinations]
http://reddit.com/r/mediaSynthesis
http://reddit.com/r/bigSleep
http://reddit.com/r/deepDream
http://reddit.com/r/ipfs

producthunt
ipfs forums
eleutherai discord
a latent space discord
4chan

envisioning slack

**Comment by gokaykucuk** [comment]
Types of texts required: 

1 - Small general description
2 - IPFS related description
3 - Detailed engaging text
4 - Some image examples

**Comment by gokaykucuk** [comment]
We are looking for,
- Authors of notebooks
- Couple of developers for frontend
- IPFS node, sponsors if possible

**Comment by voodoohop** [comment]
manifesto

**Comment by voodoohop** [comment]
https://github.com/pollinations/pollinations/issues/71
